app/styles/components/extras.scss

```diff
@@ -14,3 +14,7 @@ a.link-item {
 .leaflet-container {
   height: 300px;
 }
+
+.floating.labeled.grey.search.icon.mini.button.ui.dropdown.ember-view.upward.active.visible {
```

Comment:
Tomorrow I will change one class on the element and it'll stop working

Reply:
> Tomorrow I will change one class on the element and it'll stop working

i doing the wrong things the purpose of the issue is different so,
i will fixed